packages/trpc/server/routers/viewer/workflows/getByViewer.handler.ts

```diff
@@ -94,20 +99,45 @@ export const getByViewerHandler = async ({ ctx }: GetByViewerOptions) => {
     },
   });
 
+  const teamMemberships = user.teams.map((membership) => ({
```

Comment:
this handler has similar logic than for the one for webhooks and event types. Will get simplified, see https://github.com/calcom/cal.com/pull/9488#issuecomment-1590576938